packages/portal-proto/src/components/Modals/SetModals/SavedSets.tsx

```diff
@@ -78,17 +78,25 @@ const SavedSets: React.FC<SavedSetsProps> = ({
         id: "select",
         header: "Select",
         cell: ({ row }) => (
-          <Checkbox
-            size="xs"
-            classNames={{
-              input: "checked:bg-accent checked:border-accent",
-            }}
-            aria-label={`${row.original.setId}`}
-            {...{
-              checked: row.getIsSelected(),
-              onChange: row.getToggleSelectedHandler(),
-            }}
-          />
+          <Tooltip
+            label="Set is either empty or deprecated"
+            disabled={row.original.count !== "0"}
```

Comment:
I think it would be better to move the formatting of the count out of line 63 to the table accessor so the row.original.count here can be an int. Comparing this as a string feels off

Reply:
I just updated it. Can you please verify?

Reply:
Looks great 🎉